File: frontend/src/app/dto/Node.ts
export default class Node {
  private id: number;
  private schemeId: number;
  private xCoordinate: number;
  private yCoordinate: number;

  getId():number{
      return this.id;
      }

  setId(value:number){
      this.id=value;
      }
  getSchemeId():number{
      return this.schemeId;
      }

  setSchemeId(value:number){
      this.schemeId=value;
      }

  getXCoordinate():number{
      return this.xCoordinate;
      }

  setXCoordinate(value:number){
      this.xCoordinate=value;
      }

  getYCoordinate():number{
      return this.yCoordinate;
      }

  setYCoordinate(value:number){
      this.yCoordinate=value;
      }
}

File: src/main/java/com/example/common/entity/Node.java
package com.example.common.entity;

import javax.persistence.*;

@Entity
@Table(name = "node")
public class Node extends BaseEntity {
    @Column(nullable = false)
    private Long schemeId;
    @Column(nullable = false)
    private Long xCoordinate;
    @Column(nullable = false)
    private Long yCoordinate;

    public Long getSchemeId() {
        return schemeId;
    }

    public void setSchemeId(Long schemeId) {
        this.schemeId = schemeId;
    }

    public Long getxCoordinate() {
        return xCoordinate;
    }

    public void setxCoordinate(Long xCoordinate) {
        this.xCoordinate = xCoordinate;
    }

    public Long getyCoordinate() {
        return yCoordinate;
    }

    public void setyCoordinate(Long yCoordinate) {
        this.yCoordinate = yCoordinate;
    }
}
